doc/rst/platforms/cray.rst

```diff
@@ -201,7 +201,29 @@ Using Chapel on a Cray System
         source util/setchplenv.bash
 
 
-2) Compile your Chapel program.  For example:
+2) On HPE Cray EX systems with ``CHPL_COMM=ofi``, optionally, load the
+   Cray PMI modules::
+
+      module load cray-pmi{,-lib}
+
+   This is often not required.  Usually the default PMI modules have
+   sufficient capability to support Chapel program startup.  But under
+   certain circumstances, they do not and you will see messages of the
+   following form when you run a Chapel program:
+
+   .. code-block:: sh
+
+      [PE_0]:_pmi2_add_kvs:ERROR: The KVS data segment of <num> entries
+      is not large enough.  Increase the number of KVS entries by
+      setting env variable PMI_MAX_KVS_ENTRIES to a higher value.
```

Comment:
Just for my own clarification -- we try to set `PMI_MAX_KVS_ENTRIES` to a larger value ourselves in the launcher, right? But presumably we can't set it high enough or something in all cases?